data/openapi31-test2.yaml

```diff
@@ -24,7 +24,7 @@ components:
       required:
         - id
         - name
-        - nickname
```

Comment:
there's no nickname property, if this was intentional need to address this at the test level